src/ckanext-natcap/ckanext/natcap/templates/footer.html

```diff
@@ -1,25 +1,89 @@
-{% ckan_extends %}
+{#
+NOTE:
 
-{% block footer_lang %}
-{% endblock %}
+This template used to extend the ckan footer template, but James needed to add
+the Stanford footer and it looked and worked better with a second `footer` tag.
 
-{% block footer_content %}
-<div class="row">
-  <div class="col-md-3">
-    <a class="icon-link" href="https://naturalcapitalproject.stanford.edu/">
-      <img src="/img/natcap-logo.png" alt="The Natural Capital Project logo" />
-    </a>
+#}
+
+<footer class="site-footer" id="footer-natcap">
+  <div class="container">
+    <div class="row">
+      <div id="natcap-logo" class="col-md-3">
+        <a class="icon-link" href="https://naturalcapitalproject.stanford.edu/">
+          <img src="/img/natcap-logo.png" alt="The Natural Capital Project logo" />
+        </a>
+      </div>
+      <div class="col-md-9">
+        <h2>About</h2>
+        <div>
+          <p>
+            The Natural Capital Project Data Hub provides analysis-ready ecosystem services data as well as data for and from InVEST®, our suite of free, open-source ecosystem service models. With this Data Hub, we intend to make it faster and simpler for anyone to create and use ecosystem services information for a wide range of analyses and decisions. <a href="mailto:datahub@naturalcapitalproject.org">Contact us</a> with feedback about the site or suggestions for open access datasets to add.

```

Comment:
```suggestion
            The Natural Capital Project Data Hub provides analysis-ready ecosystem services data as well as data produced by and used in InVEST®, our suite of free, open-source ecosystem service models. With this Data Hub, we intend to simplify and accelerate the creation and use of ecosystem services information for a wide range of analyses and decisions. <a href="mailto:datahub@naturalcapitalproject.org">Contact us</a> with feedback about the site or suggestions for open access datasets to add.

```
I realize this language is not really a part of this PR but I thought I'd streamline the wording. Feel free to disregard if this doesn't align with the tone we want here!

Reply:
Looks good to me, thanks @claire-simpson !